### ISSUE TITLE
Rehike half english half polish

### ISSUE THREAD

**Issue by TheRealRysieQu** [Rehike/Rehike#365]
### Rehike version

0.8.0

### Operating system

Windows NT 6.1 build 7601 (Windows 7 Ultimate Edition Service Pack 1) AMD64

### PHP version

8.2.4

### What's going on?

So I'm from Poland and since Rehike has Polish language WIP, I switched the YouTube language to english and the country to US just to be sure it doesn't randomly get buggy or whatever like it sometimes does in other games/programs. Then I noticed that most of the text is indeed in english, but for example the player menu is completely in polish, which kinda frustrates me because it stands out from everything else.
 
![image](https://github.com/Rehike/Rehike/assets/140410551/92474930-6f02-4fbc-b773-28d6d2920e05)

And I report this as some sort of bug because when I turn off Rehike, the original YouTube player menu is in english like everything else, plus when I change my whole PC language to english and country to US, it's still in polish!! Is it tracking my IP or something? I tried using VPN as well but then Rehike completely crashes. Please, is there any way I can fix it?

**Comment by YukisCoffee** [comment]
> Is it tracking my IP or something?

YouTube is, somewhat. We find the player URLs to place in the Rehike document by requesting a standard embedded player page (for the first ever YouTube video), and then we just copy the URLs from there. We just forgot to force it to request an en-US document, so it makes the player use your default computer language (not IP address, but your browser will forward an "Accept-Language" header with your regional info nonetheless). Then we cache the result of this request for a while, so that's why just changing your computer language didn't work.

Basically, the player JS URLs have per-language variants, and we forgot to make them request your own language.

> Please, is there any way I can fix it?

- [`modules/Rehike/Player/PlayerUpdater.php:40: protected static string $sourceUrl`](https://github.com/Rehike/Rehike/blob/master/modules/Rehike/Player/PlayerUpdater.php#L40) - add to the end of this "?hl=en"
- Delete `cache` in your Rehike install directory.

**Comment by TheRealRysieQu** [comment]
@YukisCoffee oh alright, thank you so much!

**Comment by YukisCoffee** [comment]
Won't reopen, but we need to fix this properly in an update. @aubymori I don't want to stash could you do it rq?

**Comment by aubymori** [comment]
> Won't reopen, but we need to fix this properly in an update. @aubymori I don't want to stash could you do it rq?

ok lol
